tests/eas/generic.py

```diff
@@ -439,10 +444,10 @@ class RampUp(_EnergyModelTest):
                         "kind"   : "Ramp",
                         "params" : {
                             'period_ms': WORKLOAD_PERIOD_MS,
-                            "start_pct" :  5,
-                            "end_pct"   : 70,
-                            "delta_pct" :  5,
-                            "time_s"    :  2,
+                            "start_pct" : start_pct,
+                            "end_pct"   : end_pct,
+                            "delta_pct" : delta_pct,
+                            "time_s"    : 2. / ((end_pct - start_pct) / delta_pct),
```

Comment:
Maybe factor out this one as well ?

Reply:
Argh, sorry, I went too fast. I'm suggesting to factor out  the `2.` in `time_s`. You could give it a nice name like `test_duration` or something like that. That's just a nitpick though.

Reply:
Could always just give rta.Ramp a 'duration_s' argument - if it's specified then it's calculated this way.

Reply:
@bjackman By "give rta.Ramp a 'duration_s' argument", do you mean modify rta.py do accept a duration_s argument, or just declare it in the workload dict ? I don't see any 'duration_s' parameter in rta so I figured it's the former, but I want to make sure.

Reply:
Yeah modify rta.py